sycl/doc/extensions/supported/sycl_ext_intel_buffer_location.asciidoc

```diff
@@ -45,7 +45,7 @@ The use of this extension requires a target that supports SPV_INTEL_fpga_buffer_
 == Overview
 
 On targets that provide more than one type of global memory, knowing that a particular pointer can only access one of those memory types at compile time can enable compiler optimizations.  
-This extension adds an accessor property to indicate to the runtime in which of these memories the buffer corresponding to this accessor should be allocated and to inform the compiler that all accesses made through that accessor can only ever interact with the given memory.
+This extension adds an accessor property to indicate to the runtime in which of these memories the buffer corresponding to this accessor should be allocated and to inform the compiler that all accesses made through that accessor can only ever interact with the given memory or it can be used to inform the usm alocation (`malloc_device`, `malloc_shared`, `malloc_host`) to allocate on a given memory.
 This information is not a hint; it is a functional requirement of the program that must be respected.
 
```

Comment:
Can you also add a section definition the feature-test macro for this extension?  You can use the wording in the specification template in #5663.  The name of the macro should be `SYCL_EXT_INTEL_BUFFER_LOCATION`.

Since there was no feature-test macro before, the value will always be `1`.  If any further changes are made to this extension, we'll bump the feature-test macro at that time.

Of course, the implementation in #5634 should also add a definition for this macro.